composer.json

```
@@ -15,15 +15,14 @@
     ],
     "require": {
         "php": ">=8.0",
-        "psr/simple-cache": "^3",
-        "psr/cache": "^3.0"
+        "psr/simple-cache": "^3"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^v3.65.0",
         "phpunit/phpunit": "^9.6.18",
-        "squizlabs/php_codesniffer": "^3.11.1",
+        "squizlabs/php_codesniffer": "3.12.1",
```

I think it's ok to add `^` for dev dependencies. This way it accepts any new patch or minor versions, as long as it doesn't break backward compatibility.

```suggestion
        "squizlabs/php_codesniffer": "^3.12.1",
```

---

done